bin/snapshot-backups/site-snapshot-download

```diff
@@ -27,26 +38,30 @@ if [ -z "$DEPLOY_TAG" ]; then
   exit 1
 fi
 
-if [ -f $DEPLOY_TAG.zip ]; then
-  mv $DEPLOY_TAG.zip $DEPLOY_TAG.zip.bak
-fi
+if [ x$echo = x ]; then
+  if [ -f $DEPLOY_TAG.zip ]; then
+    mv $DEPLOY_TAG.zip $DEPLOY_TAG.zip.bak
+  fi
 
-mkdir $DEPLOY_TAG
-success=$?
+  mkdir $DEPLOY_TAG
+  success=$?
+else
+  success=0
+fi
 
 if [ "$success" != "0" ]; then
     echo "Unable to create directory $DEPLOY_TAG.  Does it already exist?"
     exit 1
 fi
 
-$echo source bin/cloudgov/get-s3-access storage >/dev/null 2>&1
+source bin/cloudgov/get-s3-access storage >/dev/null 2>&1
 
 echo "Downloading files from s3 to local directory './${DEPLOY_TAG}'. This could take awhile."
 echo aws s3 cp --only-show-errors s3://$S3_BUCKET/web-backup/$DEPLOY_TAG $DEPLOY_TAG --recursive
 $echo aws s3 cp --only-show-errors s3://$S3_BUCKET/web-backup/$DEPLOY_TAG $DEPLOY_TAG --recursive
 
 echo "Creating zip file ${DEPLOY_TAG}.zip"
-echo zip -rv ${DEPLOY_TAG}.zip ${DEPLOY_TAG}
+echo "zip -rv ${DEPLOY_TAG}.zip ${DEPLOY_TAG} &> $DEPLOY_TAG.zip.log"
 $echo zip -rv ${DEPLOY_TAG}.zip ${DEPLOY_TAG} &> $DEPLOY_TAG.zip.log
```

Comment:
LOL. I ran this script with --dryrun: 
`bin/snapshot-backups/site-snapshot-download --dryrun dev foo`
and I wound up with a foo.zip.log file containing "zip -rv foo.zip foo"

Reply:
OK, fixed this - thanks for finding it!